lineapy/plugins/base.py

```diff
@@ -125,7 +125,7 @@ def slice_dag_helper(
         artifacts_code = {}
         artifact_safe_names = []
         for slice_name in slice_names:
-            artifact_var = safe_var_name(slice_name)
+            artifact_var = slugify(slice_name)
```

Comment:
slugify is probably more complete than safe_var_name. I did not think about unicode inputs but people might start saving artifacts with non-ascii words and it'll probably be fine for the most part. I'm only worried about one case when slugify returns an empty string (a test case with kanji or hanzi chars should verify this). For now, if that ever happens simply raise an Exception and quit. We'll think about what to do later.